### ISSUE TITLE
Turn timeline on Our Story from image into html

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeUKGenealogyWP#72]
Image text is not readable by screen readers- doesn't meet accessibility standard

**Comment by akhilpokle** [comment]
Found this plugin : https://www.wp-timelineexpress.com/pricing/ 

Since we need only 1 timeline, I think the free version meets our needs.

**Comment by DeniseColbert** [comment]
Thanks @AKHILP96, phase 2 though, so not a priority right now.
